composer.json

```diff
@@ -1,5 +1,16 @@
 {
+    "name": "lndrest/composer",
+    "description": "An lnd class",
+    "type": "composer-plugin",
     "require": {
-        "guzzlehttp/guzzle": "^7.0"
```

Comment:
why is this removed? I think that is a dependency of that code and thus needs to be specified here.